import-dashboards.py

```diff
@@ -25,6 +26,40 @@
 OLD_VERSION_FILE = GRAFANA_DB_DIR + '/PERCONA_DASHBOARDS_VERSION'
 HOST             = 'http://127.0.0.1:3000'
 LOGO_FILE        = '/usr/share/pmm-server/landing-page/img/pmm-logo.svg'
+CONTENT          = '''<center>
+<p>MySQL and InnoDB are trademarks of Oracle Corp. Proudly running Percona Server. Copyright (c) 2006-2018 Percona LLC.</p>
+<div style='text-align:center;'>
+<a href='https://percona.com/terms-use' style='display: inline;'>Terms of Use</a> | 
+<a href='https://percona.com/privacy-policy' style='display: inline;'>Privacy</a> | 
+<a href='https://percona.com/copyright-policy' style='display: inline;'>Copyright</a> | 
+<a href='https://percona.com/legal' style='display: inline;'>Legal</a>
+</div>
+</center>
+<hr>
+<link rel='stylesheet' type='text/css' href='//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.css' />
+<script src='//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/3.0.3/cookieconsent.min.js'>
+</script>
+<script>
+function bbb(){  
+    setTimeout(function (){ 
+        window.cookieconsent.initialise(
+            {'palette': {'popup': {'background': '#eb6c44','text': '#ffffff'}, 
+                         'button': {'background': '#f5d948'}
+                        },
+             'theme': 'classic', 
+             'content': {'message': 'This site uses cookies and other tracking technologies to assist with navigation, 
+        analyze your use of our products and services, assist with promotional and marketing efforts, allow you to give feedback, 
```

Comment:
Could you please double check whether this does not raise javascript error, because of unsnapped newlines in javascript code.

Reply:
Newlines have been removed in the javascript code.